modules/javafx.graphics/src/main/java/javafx/scene/TraversalDirection.java

```diff
@@ -22,20 +22,24 @@
  * or visit www.oracle.com if you need additional information or have any
  * questions.
  */
-
-package com.sun.javafx.scene.traversal;
+package javafx.scene;
 
 /**
- * Specifies the traversal method.
+ * Specifies the direction of focus traversal.
+ *
+ * @since 24
  */
-public enum TraversalMethod {
-    /**
-     * Traversal was initiated programmatically or by clicking.
-     */
-    DEFAULT,
-
-    /**
-     * Traversal was initiated by pressing a key on the keyboard.
-     */
-    KEY
+public enum TraversalDirection {
+    /** Moves focus downward. */
```

Comment:
The direction itself doesn't move the focus. I would use a different phrasing, maybe something like "Indicates a focus change to the node below the currently focused node".

Same with the others.

Reply:
good point!